tests/integration/delete.bats

```diff
@@ -89,9 +89,10 @@ EOF
 
     for s in ${subsystems}; do
       name=CGROUP_${s^^}
-      eval path=\$${name}/foo
-      echo $path
-      [ -d ${path} ] || fail "test failed to create memory sub-cgroup"
+      eval path=\$"${name}"/foo
+      # shellcheck disable=SC2154
+      echo "$path"
```

Comment:
Can we remove this echo?

Reply:
Looks like we can, but the warning (and thus the `shellcheck disable=` still remains since shellcheck don't grok eval (https://github.com/koalaman/shellcheck/wiki/SC2034#indirection).

Updated